### PR TITLE
Feature review

### PR DESCRIPTION
**Pull request by Obote**
Hello! :smiley: 
Great work and effort you have put into this project so far.

### Highlights :+1: 
Good use of Semantic HTML concepts. :100: 
Descriptive commit messages. :100: 
However, there is always room for improvement. Your long hours of work are very much appreciated. However, the code review process is for improving your code, not for criticizing it.

### Changes Required :recycle: 
Please check the comments under the review.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+
```
Comment (Obote) [inline index.html:3]: Kindly remove the extra blank lines and indentations for a cleaner code file.
Please keep in mind that we're only adding blank lines to separate two logical pieces of code or two large sections apart. This brings confusion to other programmers or you in the future.

Comment (Obote) [inline index.html:3]: Kindly remove the extra blank lines and indentations for a cleaner code file.
Please keep in mind that we're only adding blank lines to separate two logical pieces of code or two large sections apart. This brings confusion to other programmers or you in the future.

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+  display: flex;
+}
+#nav-bar ul li {
+  padding: 1rem 2rem;
+}
+#nav-bar ul li a {
+  color: #333;
+  font-size: 1.2rem;
+}     } */
+
```
Comment (Obote) [inline styles.css:50]: Please remove out commented code in your codebase. This makes code unclean and brings confusion to other programmers or you in the future.

File: index.html (hunk 1 above)
Comment (Obote) [inline index.html:3]: Kindly remove the extra blank lines and indentations for a cleaner code file.
Please keep in mind that we're only adding blank lines to separate two logical pieces of code or two large sections apart. This brings confusion to other programmers or you in the future.

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,252 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+
+  /* //display: flex;
+  //background-color: #ff8c00; */
```
Comment (Obote) [inline styles.css:7]: Please remove out commented code in your codebase. This makes your code unclean and brings confusion to other programmers or you in the future.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+<body>
+  <header id="header">
+    <nav id="nav-bar">
+      <img id="header-img" src="https://images-na.ssl-images-amazon.com/images/I/51uTvI7ew-L._AC_SY450_.jpg" alt="logo" class="logo">      
+      <a class="nav-link" href="#features">Features</a>
+      <a class="nav-link" href="#how-it-work">How It Works</a>
+      <a class="nav-link" href="#pricing">Pricing</a>      
+    </nav>
+  </header>
+
```
Comment (Obote) [inline index.html:22]: Kindly remove the extra blank lines and indentations for a cleaner code file.
Please keep in mind that we're only adding blank lines to separate two logical pieces of code or two large sections apart. Also, the spaces bring linters errors.

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @Obote 😃, 

Great review you performed out there, but please keep in mind that you should submit the right link to the PR, and not the page for the files changed. 🙏🏻
Please notice that this mistake could invalidate your submission 😊
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Obote** [comment]
Thanks for the feedback, I will keep it in mind to avoid making the same mistake next time.

**Pull request merged**